node/src/actor_system_factory.rs

```diff
@@ -1089,8 +1089,9 @@ mod tests {
             payment_thresholds_opt: Some(PaymentThresholds::default()),
             when_pending_too_long_sec: DEFAULT_PENDING_TOO_LONG_SEC,
         };
-        let persistent_config =
-            PersistentConfigurationMock::default().chain_name_result("eth-ropsten".to_string());
+        let persistent_config = PersistentConfigurationMock::default()
+            .chain_name_result("eth-ropsten".to_string())
```

Comment:
You'll do better if you chose a different chain. Otherwise you can add work for other people who will finally decide to tear all ropsten code out.

Reply:
Doing that will cause a panic, also it was used like that before I modified.

Reply:
Ok